1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,6 +1,22 @@
-function setAlarm() {}
+function setAlarm() {
+const timeRemaining = document.getElementById("timeRemaining")
+const setAlarmInput = document.getElementById("alarmSet")
+let takingInput = Number(setAlarmInput.value)
```

Comment:
I think the task asked you to format the time like a digital clock, so 00:00, rather than 000 secs